src/main/java/org/nguh/nguhcraft/data/NguhcraftRecipeGenerator.kt

```diff
@@ -235,6 +241,11 @@ class NguhcraftRecipeGenerator(
         offerStonecuttingFamily(NguhBlocks.CALCITE_BRICK_FAMILY, Blocks.CALCITE)
         offerStonecuttingRecipe(RecipeCategory.BUILDING_BLOCKS, NguhBlocks.PYRITE_BRICKS, NguhBlocks.PYRITE)
 
+        // =========================================================================
+        //  Smelting
+        // =========================================================================
+        offerSmelting(Items.BONE_BLOCK, RecipeCategory.BUILDING_BLOCKS, Items.CALCITE, 0.2, 200, "nguhStoneRenewables")
```

Comment:
```suggestion
        offerSmelting(Items.BONE_BLOCK, Items.CALCITE)
```
And add this as a helper function to this class:
```kotlin
    fun offerSmelting(Input: ItemConvertible, Output: ItemConvertible, Experience: Float = .2f)
        = offerSmelting(listOf(Input.asItem()), RecipeCategory.MISC, Output.asItem(), Experience, 200, null)
```